extension/src/popup/styles/global.scss

```diff
@@ -3,6 +3,7 @@
   --z-index--loader: -1;
   --z-index--scam-warning: 20;
   --z-index--ledger: 30;
+  --z-index-account-list: 40;
```

Comment:
this was needed now that the ledgerSign component is used on the /account page. So had to make that parent the same z-index level as ledger, and make the account-list z-index above it